svy/src/helpers/account.ts

```diff
@@ -0,0 +1,12 @@
+import { Address } from "@graphprotocol/graph-ts";
+import { Account } from "../../generated/schema";
+
+export function getOrCreateAccount(address: Address): Account {
+  const id = address.toHexString();
+  let account = Account.load(id);
+  if (account === null) {
+    account = new Account(id);
+    account.save();
```

Comment:
When creating new entities you must provide a default value for all required and non-derived fields